src/lavague/driver.py

```diff
@@ -1,16 +1,22 @@
 from abc import ABC, abstractmethod
 from typing import Any
 from selenium.webdriver.remote.webdriver import WebDriver
+from playwright.sync_api import Page
 
 
 class AbstractDriver(ABC):
-    def getDriver(self) -> Any:
+    def getDriver(self) -> tuple[str, Any]:
+        """Return the expected variable name and the driver object"""
         pass
 
     @abstractmethod
     def getUrl(self) -> str:
         pass
 
+    @abstractmethod
+    def getUrlCode(self, url: str) -> str:
```

Comment:
Not important, but the name is little confusing a better name would be goToUrlCode as we have a function with name gotourl and this function returns the code for that.

Reply:
you're right